### PR TITLE
fix: fix model of the last device deployment status request

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline

File: app/app.go [hunk 1]
```diff
@@ -1548,7 +1548,13 @@ func (d *Deployments) UpdateDeviceDeploymentStatus(ctx context.Context, deployme
...
 	if !ddState.Status.Active() {
 		l := log.FromContext(ctx)
-		if err := d.db.SaveLastDeviceDeploymentStatus(ctx, *dd); err != nil {
+		ldd := model.DeviceDeployment{
+			DeviceId:     dd.DeviceId,
+			DeploymentId: dd.DeploymentId,
+			Id:           dd.Id,
+			Status:       ddState.Status,
+		}
+		if err := d.db.SaveLastDeviceDeploymentStatus(ctx, ldd); err != nil {
```
Comment (merlin-northern) [inline app/app.go:1557]: this is already happening in enterprise, smart to put it here up front.

**Review by merlin-northern — APPROVED** [review]
lgtm

File: app/app.go (hunk 1 above)
Comment (kjaskiewiczz) [reply app/app.go:1557]: previously we were using just *dd here, so we didn't have last status

**Pull request merged**